samples/client/petstore/typescript-fetch/builds/es6-target/runtime.ts

```diff
@@ -209,17 +209,9 @@ export function querystring(params: HTTPQuery, prefix: string = ''): string {
             }
             return `${encodeURIComponent(fullKey)}=${encodeURIComponent(String(value))}`;
         })
-        .filter(part => part.length > 0)
         .join('&');
 }
 
-export function mapValues(data: any, fn: (item: any) => any) {
```

Comment:
please run `mvn clean` before re-generating the samples.
the `mapValues` function actually is available in your version of 
https://github.com/OpenAPITools/openapi-generator/blob/73ac4482ea97e7210aefceda22aee82885612143/modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache#L207-L212
but it is removed in the generated samples, which indicates that the version you use for generating the samples is not the one you want to merge...

Reply:
Wow. That seems to have un-removed `mapValues`, but the `body` parameter was renamed back to `pet` again. My code seems to be haunted.

Reply:
the `body` to `pet` renaming could be due to general changes, not necessarily related to your code.